doc/modules/outlier_detection.rst

```diff
@@ -246,6 +246,9 @@ This strategy is illustrated below.
   (:class:`covariance.MinCovDet`) of location and covariance to
   assess the degree of outlyingness of an observation.
 
+* See :ref:`sphx_glr_auto_examples_applications_plot_outlier_detection_wine.py` for an example of robust covariance estimation
+on a real data set.
```

Comment:
Could you please wrap the line to <88 characters?

Reply:
By wrap to <88 do you mean reduce the number of characters to 87 or move everything onto the same line?